### PR TITLE
Update to 2.0.0-beta3

### PR DESCRIPTION
**Pull request by shrishrirang**
- Updated MobileServices.Web.js
- Updated description, version and dependencies in plugin.xml

### CODE REVIEW

**Comment by azurecla** [comment]
Hi **@shrishrirang**, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (shrirs). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by shrishrirang** [comment]
Non en-us resources need to be updated for the newly added string. Not sure what the procedure is.

File: plugin.xml [hunk 1]
```diff
@@ -4,16 +4,16 @@
...
-    <name>Windows Azure Mobile Services</name>
-    <description>Exposes JavaScript api to access an Azure Mobile Apps instance</description>
+           id="cordova-plugin-ms-azure-mobile-apps"
+      version="2.0.0-beta3">
+    <name>Azure Mobile Apps</name>
+    <description>This library enables you to develop scalable and secure cordova mobile apps hosted in Microsoft Azure. You can incorporate structured storage, user authentication and push notifications in minutes. For more information, check out http://azure.microsoft.com/en-us/documentation/articles/app-service-mobile-value-prop-preview</description>
     <license>Apache 2.0</license>
     <keywords>azure, services, zumo, cloud</keywords>
     <repo>https://github.com/azure/azure-mobile-services-cordova/</repo>
     <issue>https://github.com/azure/azure-mobile-services/issues</issue>
```
Comment (phvannor) [inline plugin.xml]: this should go to apps-js-client

Comment (shrishrirang) [inline plugin.xml]: Didn't quite understand the comment.. ?

Comment (phvannor) [inline plugin.xml]: This "<issue>https://github.com/azure/azure-mobile-services/issues</issue>"  should now refer to the issues list on the new js client repo

Comment (shrishrirang) [inline plugin.xml]: The comment looks obvious now :) I think I was too distracted by the red/green diffs to look at the line you commented on!

Comment (shrishrirang) [inline plugin.xml]: This is fixed in the updated PR.

**Comment by phvannor** [comment]
:+1:

**Pull request merged**